docker-compose.yml

```diff
@@ -189,12 +189,12 @@ services:
     environment:
       SPA_PATH: /openmrs/spa
       API_URL: /openmrs
-      SPA_CONFIG_URLS:
+      SPA_CONFIG_URLS: /openmrs/spa/ozone/ozone-frontend-config.json
```

Comment:
@enyachoke what does this actually _do_?

Reply:
The file itself is a general O3 frontend config file but in this case, it is actually adding the HIS component links. I got the message for the ticket but it looks okay to me.